### ISSUE TITLE
Should we enable github discussions on this repo?

### ISSUE THREAD

**Issue by andrewm4894** [LAION-AI/Open-Assistant#334]
Github discussions is actually pretty good. https://docs.github.com/en/discussions

Needs to be enabled at repo level.

It might lower barrier to people who might be a bit scared or afraid to open an issue (even though its pretty chill right now and i dont see any scary issue templates) and maybe don't want to use discord. Also could help avoid having discussions go into discord blackhole when they may end up being useful in future for others who may come. 

Just a suggestion, dont feel super strong or anything - we use a discord and gh discussions at netdata. For example I've found it really useful for just developing ML ideas in GH discussions and then they can sometimes graduate or get converted into GH issues for actual work to be done. https://github.com/netdata/netdata/discussions?discussions_q=label%3Aarea%2Fml

**Comment by Shtoner** [comment]
I'd say it's a good idea just because of the Discord blackhole like you said. I think it will allow new devs to be able to look through past discussions that they might have missed on Discord.

**Comment by clstaudt** [comment]
Please do. I may not appreciate all the benefits of Discord enough, but if the code is on GitHub, I don't really see the point of having the conversation on separate platforms.

**Comment by bitplane** [comment]
Not entirely on-topic for this issue, but if we're looking for maximum exposure to other devs, it'd be cool to have a public channel that links Discord to libera.chat.

Libera is where the official chat rooms for most of the technologies we're using (#python, #docker, #node.js, #vscode) and some of the more popular unofficial ones too (#linux, #workingset, #git). It might be a decent experiment to draw a few more devs in.

This thing looks like it can do it:

https://github.com/42wim/matterbridge

**Comment by andrewm4894** [comment]
@yk @andreaskoepf i think we maybe should do this and sooner rather than later or at least no harm in trying it out.

for example we could easily create a discussion thread for each common FAQ (using labels etc) - example being running `docker-compose` vs `docker compose` as first candidate we can just point users at.

thoughts or any objections?

mainly just thinking sooner we enable it sooner we can start sort of crowdsourcing a knowledge base source (for faq or common getting started type questions) that might help cut down on having to answer same question multiple times and things like that. 

If we were to decide to go with it then we could also add links in docs to mention it too.

apologies for being pushy, just think could help save some time and/or share some load.

**Comment by yk** [comment]
the worry is that we add another place where conversations happen and thus diverge even more. given that we already run a very decentralized operation, we have to keep conversations together: discord for chatter, github issues for tasks & bugs, and `docs/` for documentation. I feel these cover enough ground that I'm not sure adding yet another modality is a positive change. Someone running into the `docker-compose` issue can be linked to a solved past issue, or we make a paragraph in `docs/` or in the README, and then link to that.

@andreaskoepf what do you think?

**Comment by andrewm4894** [comment]
Yeah that's one risk potentially. Discord is a very acquired taste I have to say (maybe I am not cool enough :) ). But if it's working fine maybe no need change anything. More so if you find same random time sucking conversations coming up again and again as new people come into discord or gets too messy as it grows then something like GH discussion's is useful place to have conversations in open for everyone to see, future people, without polluting GH issues with lots of random stuff. I kinda find it very hard to be away from discord for a day and get back up to speed, I end up just missing lots as it's all just a big wall of text and hard to get the headlines from what I've missed.

But agree adding yet another tool might not help and maybe could just be another source of noise.

Maybe for now one option could be to just start building an FAQ section in docs somewhere and we can see if that can help as a sort of place to just put common questions or common issues people face etc. I'd be happy to pull something together in there instead if we'd rather.

**Comment by Majdoddin** [comment]
Conversatoins is a greate feature from my experience with [openai/whisper](https://github.com/openai/whisper). 
Speically "Show and Tell": people can share how they are using/adapting the repo. 
Even we can have a subtopic for "prompt engineering", for example "how can I have openAssistant to translate an SRT file properly?"
